common/firmware/head-ff/spi/spi.c

```diff
@@ -0,0 +1,243 @@
+#include "common/firmware/spi.h"
```

Comment:
Can we just have this in the `head` directory? i don't think we need to retain support for the `AxisController` development platform boards.